### ISSUE TITLE
Square yellow color when #FFFFFF but working when #ffffff

### ISSUE THREAD

**Issue by geeky3** [ggordan/GutterColor#119]
When i type "#FFFFFF" it gives me annoying yellow box but if I type "#ffffff" than it works perfectly, also uppercase with any other color works, I have issue only with '#FFFFFF". Any idea what is causing this ??

**Comment by TeralGoe** [comment]
Yep, I noticed that, too. Looks like this is happening because GutterColor have #ffffff.png in the cache folder and cannot create a new file with same name.

However, you can recreate all files through clearing cache: 
`Command Palette -> GutterColor: Clear Cache`

**Comment by marandac** [comment]
Applied a quickfix to this problem by forcing lowercase on the color strings in the line.py code (in Win10, \User\Your Username\AppData\Roaming\Sublime Text 3\Packages\Gutter Color\).

Changed those instances:
```python
if matches:
        return matches.group(0)
```

to

```python
if matches:
        lowercase = matches.group(0)
        lowercase = lowercase.lower()
        return lowercase
```

Once you update the file and clear your cache as described by @TeralGoe , the problem should be gone.

As reference, I attached the modified line.py file
[linepy.zip](https://github.com/ggordan/GutterColor/files/4915981/linepy.zip)
